languageserver/server/server.go

```diff
@@ -2061,6 +2067,20 @@ func (s *Server) getDiagnosticsForParentError(
 
 // parse parses the given code and returns the resultant program.
 func parse(code, location string, log func(*protocol.LogMessageParams)) (*ast.Program, error) {
+	defer func() {
```

Comment:
Nice. I guess we could send a Sentry request here in the future?

Reply:
will add that in yes

Reply:
Chnaged to sentry recover with context as that is mostly what we want to do here.

Reply:
Nice!